geode-core/src/main/java/org/apache/geode/cache/client/internal/ClientSideHandshakeImpl.java

```diff
@@ -234,7 +234,8 @@ public ServerQueueStatus handshakeWithServer(Connection conn, ServerLocation loc
       // Read delta-propagation property value from server.
       // [sumedh] Static variable below? Client can connect to different
       // DSes with different values of this. It shoule be a member variable.
-      if (!communicationMode.isWAN() && currentClientVersion.compareTo(Version.GFE_61) >= 0) {
+      if (!communicationMode.isWAN() && currentClientVersion.compareTo(
```

Comment:
I've seen a number of these changes where we've added a newline somewhere in the middle of an expression...not sure if you are doing these intentionally or your IntelliJ formatter has gone rogue.

Reply:
I think that happened when I was trying to subclass Version to keep something in org.apache.geode.internal that could refer to all of its statics.  I gave up on that and undid the change but IJ seems to have left some formatting turds behind.